core/src/pixelcolor/raw/to_bytes.rs

```diff
@@ -153,6 +153,38 @@ mod tests {
         assert_all_orders(Gray8::new(255), [255]);
     }
 
+    #[test]
+    fn bpp12_rgb_be() {
+        assert_eq!(
+            Rgb444::new(15, 0, 0).to_be_bytes(),
+            [0b0000_1111, 0b000_00000]
+        );
+        assert_eq!(
+            Rgb444::new(0, 15, 0).to_be_bytes(),
+            [0b0000_0000, 0b1111_0000]
+        );
+        assert_eq!(
+            Rgb444::new(0, 0, 15).to_be_bytes(),
+            [0b0000_0000, 0b0000_1111]
+        );
+    }
+
+    #[test]
+    fn bpp12_rgb_le() {
+        assert_eq!(
+            Rgb444::new(0, 0, 15).to_le_bytes(),
+            [0b0000_1111, 0b0000_0000]
+        );
+        assert_eq!(
+            Rgb444::new(0, 15, 0).to_le_bytes(),
+            [0b1111_0000, 0b0000_0000]
+        );
+        assert_eq!(
+            Rgb444::new(15, 0, 0).to_le_bytes(),
+            [0b0000_0000, 0b0000_1111] // not ideal, a side effect of using 16 bits to store 12 bits
+        );
+    }
+
```

Comment:
I don't think this is needed when `Rgb444` is stored as a `Raw16`. This should be covered by the bit positions test in `rgb_color.rs`.

```suggestion
```

Reply:
Fair enough!